_config.yml

```diff
@@ -34,6 +34,7 @@ default_js:
   - "https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.js"
   - "/scripts/search.js"
   - "/scripts/clipboard.js"
+  - "/scripts/open-in-repl.js"
```

Comment:
To avoid scanning and modifying the same code blocks twice, we may want to consider combining clipboard/open-in-repl?